changelog/unreleased/changes/2417--aggregation-function-plugins.md

```diff
@@ -0,0 +1,6 @@
+Aggregation functions in the `summarize` operator are now plugins, which makes
+them easily extensible. The syntax of `summarize` now supports specification of
+output field names, similar to SQL's `AS` in `SELECT f(x) AS name`.
```

Comment:
@mavam we should also mention that `summarize` is no longer a plugin, but rather a built-in operator. As such, it is no longer versioned separately and does not need to be enabled explicitly in the configuration anymore.

Reply:
Ah, right. Feel free to adjust the wording.